apps/health_campaign_field_worker_app/lib/utils/extensions/extensions.dart

```diff
@@ -16,7 +16,7 @@ part 'context_utility.dart';
 
 extension DateAgeExtension on DateTime {
   int get age {
-    return (DateTime.now().difference(this).inDays / 365).round();
+    return (DateTime.now().difference(this).inDays / 366).round();
```

Comment:
This wont fix the issue we need to to .floor (DateTime.now().difference(this).inDays / 366).floor(). As when we enter age directly we are setting date as 01-01, and if the current month is july or later the current age will be incremeneted by 1 as we are rounding it off